database/src/main/scala/cromwell/database/sql/tables/WorkflowStoreEntry.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.database.sql.tables
+
+import java.sql.Timestamp
+
+case class WorkflowStoreEntry
+(
+  workflowUuid: String,
+  workflowSource: String,
+  workflowInputs: Option[String],
+  workflowOptions: Option[String],
```

Comment:
The source, inputs, and options should be `Clob`, just like in ye olde worlde [[1](https://github.com/broadinstitute/cromwell/blob/d16ee8d3e457baaf5334fc79045d2d98d96a2823/database/src/main/scala/cromwell/database/slick/WorkflowExecutionAuxComponent.scala#L15-L17)] [[2](https://github.com/broadinstitute/cromwell/blob/d16ee8d3e457baaf5334fc79045d2d98d96a2823/database/src/main/scala/cromwell/database/obj/WorkflowExecutionAux.scala#L8-L10)].

They can still be nullable if you'd like, you'll just need to wrap them as an `Option[Clob]`.